site/content/integrate/apps/quickstart/apps-debug/_index.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Using /apps debug command"
+heading: "Using /apps debug command"
+description: "`/apps debug` command can be used to debug an App's bindings, KV store records, view the logs, etc."
+weight: 100
+aliases:
+  - /integrate/apps/apps-debug/
+---
+
+This reference guide documents /apps debug command that can be used to display
```

Comment:
The /apps debug command is available to help developers debug problems with their apps.  This slash command can be used to provide debug logs right in a Mattermost channel, examine, change and delete data stored by your app and examine OAuth2 data.

This command is available exclusively when a Mattermost server is running developer mode.  Instructions for turning on developer mode are available <here> **TODO**

Reply:
The rest about the actual slash commands is great.  This is so cool!